docs/guides/operator-guide-aws-marketplace.md

```diff
@@ -352,20 +352,20 @@ The following table includes some additional commands that might help you manage
 
 ## UID2 Operator Error Codes
 
-The following table lists possible errors while setting up a Private Operator.
+The table below lists potential errors during a Private Operator's startup sequence.
 
 :::note
-This applies only to Private Operators starting from Q2 2024.
+Only Private Operators versions released on and after Q2 2024 will present these error codes.
```

Comment:
Suggest:
Error codes for Private Operator startup issues apply only to versions released in Q2 2024 and later.